content/docs/user-guide/basic-concepts.md

```diff
@@ -0,0 +1,129 @@
+# Basic Concepts
+
+Intro and DVC philosophy...possible diagram of cache/remote/workspace
+
+## Cache
+
+_From `dvc cache`_
```

Comment:
Thanks for these notes. Could be `<!-- HTML comments -->` for now if you want.

Reply:
Or actually, if you can list in the description of the PR the list of initial concepts you intend to put here and where they exist currently, that would be idea.

Reply:
p.s. If there are any that you think should also be included but don't have a main place in existing docs let's list them too but we can address them in a future PR. This one should be mostly about concentrating existing content to the new doc/section, and link whatever is needed (for SEO purposes).

Reply:
I browsed and thought on this for a bit and didn't find any obvious candidates. Listed Project as a possible opportunity. `params` is a good one, but it's already in the glossary (and now as a concept.)